server.js

```diff
@@ -7620,14 +7620,20 @@ cache({
         if (err != null || !res || res.statusCode !== 200)
           throw 'inaccessible';
 
+        badgeData.colorscheme = 'brightgreen';
+
         switch (type){
           case 'json':
             data = (typeof data == 'object' ? data : JSON.parse(data));
-            badgeData.text[1] = (prefix || '') + jp.query(data, pathExpression).join(', ') + (suffix || '');
+            var jsonpath = jp.query(data, pathExpression);
+            if (!jsonpath.length)
```

Comment:
For fewer surprises, would it be better to use `Array.isArray`?

Reply:
From some testing it seems jsonpath will always return an array,
If the query finds nothing, jsonpath will just return an empty array `[]`

examples based on the following json
```javascript
{
  name: 'project',
  version: '1.2.3'
}
```
`?query=` | jsonpath returned
:--: | :--:
`$.version` | `['1.2.3']`
`$.notfound` | `[]`